t/udp-generator.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
+
+# Read datagrams from input files and send them to a specified server
+
+use strict;
+use IO::File;
+use IO::Socket;
+
+if (scalar @ARGV < 3) {
+	print STDERR "Usage: $0 [server] [port] [input...]\n";
+	exit 1;
+}
+
+my $server = shift @ARGV;
+my $port = shift @ARGV;
+
+my $sock = IO::Socket::INET->new(
+	Type     => SOCK_DGRAM,
+	Proto    => 'udp',
+	PeerAddr => $server,
+	PeerPort => $port,
+) or die "Failed to create a UDP socket: $!\n";
+
+foreach my $f (@ARGV) {
```

Comment:
```suggestion
for my $f (@ARGV) {
```

"foreach" is a synonym of "for" and is rarely used today.